### PR TITLE
BTAT-993, BTAT-1001, BTAT-1016

### PR DESCRIPTION
**Pull request by lewishayqa**

### CODE REVIEW

**Comment by miranda-hawkes** [comment]
Please fix merge conflicts :)

**Comment by lewishayqa** [comment]
This has been rebased now and conflicts are fixed :)

File: build.sbt [hunk 1]
```diff
@@ -45,6 +69,15 @@ def test(scope: String = "test, it"): Seq[ModuleID] = Seq(
...
+    Group(
+      test.name,
+      Seq(test),
+      SubProcess(ForkOptions(runJVMOptions = Seq("-Dtest.name=" + test.name, "-Dlogger.resource=logback-test.xml")))
+    )
+}
+
 lazy val microservice = Project(appName, file("."))
   .enablePlugins(Seq(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin) ++ plugins : _*)
   .settings(scalaSettings: _*)
```
Comment (miranda-hawkes) [inline build.sbt]: You need to add `.settings(coverageSettings: _*)` and `.settings(playSettings: _*)` here in order to apply your new settings

File: build.sbt [hunk 2]
```diff
@@ -19,15 +19,39 @@ import uk.gov.hmrc.SbtAutoBuildPlugin
...
+    ScoverageKeys.coverageMinimum := 90,
+    ScoverageKeys.coverageFailOnMinimum := false,
+    ScoverageKeys.coverageHighlighting := true
+  )
+}
 
 val compile = Seq(
   ws,
-  "uk.gov.hmrc" %% "frontend-bootstrap" % "8.6.0",
+  "uk.gov.hmrc" %% "frontend-bootstrap" % "8.9.0",
```
Comment (miranda-hawkes) [inline build.sbt]: Can you change this to 8.10.0 please?

**Pull request merged**